keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -143,14 +143,33 @@ be extended to cover the new behavior if enabled.
 
 #### Alpha
 
-- [ ] Implement the new feature gate and kubelet configuration
-- [ ] Ensure proper tests are in place
-- [ ] Update documentation to make the feature visible
+- [x] Implement the new feature gate and kubelet configuration
+- [x] Ensure proper tests are in place
+- [x] Update documentation to make the feature visible
 
 #### Alpha to Beta Graduation
 
-- [ ] Enable the feature per default
-- [ ] No major bugs reported in the previous cycle
+- [x] Enable the feature per default
+- [x] Making the feature API aware
+- [x] No major bugs reported in the previous cycle
+
+##### API Awareness
+
+We add conversion methods to the API Server with the graduation of the feature
+to beta. The beta graduation does not mean that it's enabled by default, which
+got introduced in Kubernetes v1.24.
+
+The API Awareness allows us giving users direct feedback about the chosen seccomp
+profile as well as handling upgrade scenarios. If the feature is enabled, then
+created workloads not specifying any `securityContext.seccompProfile.type` will
+default to `RuntimeDefault`.
```

Comment:
What are the implications for defaulting securityContext and seccomp profile for workloads which do not specify an operating system, but are scheduled to windows notes? At best that seems misleading, at worst does it get rejected by the windows kubelet?

Reply:
Looking at the [Windows docs](https://kubernetes.io/docs/concepts/windows/intro/) it seems that seccomp is supported by the currently available container runtimes. So I would not distinguish the field by the host OS or architecture to keep it simple. It could be still possible that seccomp is not supported from the Kernel or build time perspective. Container runtimes should then ignore the field and unset it in the OCI spec (which is the current behavior).